app/components/views/GetStartedPage/PrivacyPage/CustomPrivacyOptions.jsx

```diff
@@ -17,7 +17,7 @@ export default ({ toggleCustomPrivacy, acceptCustomPrivacy, ...props }) => (
       />
     </div>
     <div className={styles.customPrivacySettings}>
-      <PrivacySettings {...props} />
+      <PrivacySettings {...{ ...props, showChangePassPhraseOption: false }} />
```

Comment:
Thanks, but we already have a props which can be used for that. `walletReady`

Reply:
@vctt94 Great! That's good to know. I'll fix it.